src/Microsoft.ML.TimeSeries/SsaSpikeDetector.cs

```diff
@@ -26,8 +26,7 @@
 namespace Microsoft.ML.Transforms.TimeSeries
 {
```

Comment:
please also clean up the xml for the 4 estimator classes: SsaSpikeEstimator, etc. Their xml shouldn't have any samples and just cref to extension method APIs for usage, plus include algorithm details, in this case link to SSA paper or IID wikipage.